### PR TITLE
AAC template revisions for consistency of pillar capitalization

### PR DESCRIPTION
**Pull request by ShannonLeavitt**
This PR adjusts WAF pillar names for consistency of capitalization.

### CODE REVIEW

File: templates/org/aac/aac-architecture-guide-content.md [hunk 1]
```diff
@@ -44,12 +44,19 @@ Other contributors: > Include contributing (but not primary) authors, major edit
 
 ## Next steps
```
Comment (ShannonLeavitt) [inline templates/org/aac/aac-architecture-guide-content.md:45]: The purpose of this change is to resolve confusion about what kind of content the two final sections should include. One of the Aquent editors brought up this issue. To resolve the confusion, I copied the content from the aac-article-title-content.md template, where the purpose of the sections is presumably the same.

**Comment by ShannonLeavitt** [comment]
@ckittel - Can you review this PR and sign off if you approve?

File: templates/org/aac/aac-article-title-content.md [hunk 2]
```diff
@@ -153,15 +153,15 @@ A deployment for a reference architecture that implements these recommendations
 
 Principal authors: > Only the primary authors. Listed alphabetically by last name. Use this format: Fname Lname. If the article gets rewritten, keep the original authors and add in the new one(s).
 
- - [Author 1 Name](http://linkedin.com/ProfileURL) | Title, such as "Cloud Solution Architect"
- - [Author 2 Name](http://linkedin.com/ProfileURL) | Title, such as "Cloud Solution Architect"
- - > Continue for each primary author (even if there are 10 of them).
+- [Author 1 Name](http://linkedin.com/ProfileURL) | Title, such as "Cloud Solution Architect"
+- [Author 2 Name](http://linkedin.com/ProfileURL) | Title, such as "Cloud Solution Architect"
```
Comment (ckittel) [inline templates/org/aac/aac-article-title-content.md]:
```suggestion
- [Author 1 Name](https://linkedin.com/in/ProfileURL) | Title, such as "Cloud Solution Architect"
- [Author 2 Name](https://linkedin.com/in/ProfileURL) | Title, such as "Cloud Solution Architect"
```

File: templates/org/aac/aac-article-title-content.md [hunk 3]
```diff
@@ -153,15 +153,15 @@ A deployment for a reference architecture that implements these recommendations
...
+- > Continue for each primary author (even if there are 10 of them).
 
 Other contributors: > Include contributing (but not primary) authors, major editors (not minor edits), and technical reviewers. Listed alphabetically by last name. Use this format: Fname Lname. It's okay to add in newer contributors.
 
- - [Contributor 1 Name](http://linkedin.com/ProfileURL) | Title, such as "Cloud Solution Architect"
- - [Contributor 2 Name](http://linkedin.com/ProfileURL) | Title, such as "Cloud Solution Architect"
- - > Continue for each additional contributor (even if there are 10 of them).
+- [Contributor 1 Name](http://linkedin.com/ProfileURL) | Title, such as "Cloud Solution Architect"
+- [Contributor 2 Name](http://linkedin.com/ProfileURL) | Title, such as "Cloud Solution Architect"
```
Comment (ckittel) [inline templates/org/aac/aac-article-title-content.md]:
```suggestion
- [Contributor 1 Name](https://linkedin.com/in/ProfileURL) | Title, such as "Cloud Solution Architect"
- [Contributor 2 Name](https://linkedin.com/in/ProfileURL) | Title, such as "Cloud Solution Architect"
```

File: templates/org/aac/aac-architecture-guide-content.md [hunk 4]
```diff
@@ -44,12 +44,23 @@ Other contributors: > Include contributing (but not primary) authors, major edit
...
-- Learn links should be site-relative, for example (/azure/feature/article-name).
-- Don't include trailing slash in any links.
+> Link to Learn articles. Could also be to appropriate sources outside of Learn, such as GitHub repos, third-party documentation, or an official technical blog post. 
+> - Links shouldn't include en-us locale unless they don't work without it.
+> - Learn links should be site-relative, for example (/azure/feature/article-name).
+> - Don't include trailing slash in any links.
+> Example:
+
+- [Azure Machine Learning documentation](/azure/machine-learning)
+- [What are Azure Cognitive Services?](/azure/cognitive-services/what-are-cognitive-services)
```
Comment (ckittel) [inline templates/org/aac/aac-architecture-guide-content.md]:
```suggestion
- [What are Azure AI Services?](/azure/ai-services/what-are-ai-services)
```

**Review by ckittel — APPROVED** [review]
Looks good @ShannonLeavitt.  Just a few small suggestions, but nothing blocking.  The `https` thing has been on my list for a while now, we still get contributions with people having `http` due to copy and paste.

**Comment by ShannonLeavitt** [comment]
@ckittel - I've now fixed those links across all of the template files.

**Comment by ShannonLeavitt** [comment]
#sign-off

**Comment by ckittel** [comment]
Attn: @mattbriggs

**Review by mattbriggs — APPROVED** [review]

**Pull request merged**